NEWS.rst

```diff
@@ -6,7 +6,7 @@ every change, see the Git log.
 
 Latest
 ------
-* tbd
+* Major: Changed the metric_flags to a metric_kind representing whether the metric is a counter, constant or a gauge.
```

Comment:
Add something like:
* Major: Changed the protocol. New protocol version is 1.

That way we can keep track of the protocol version and library version.